lib/mongo/operation/write/create_index.rb

```diff
@@ -42,37 +42,14 @@ module Write
       #
       # @since 2.0.0
       class CreateIndex
-        include Executable
+        include GLEable
+        include WriteCommandEnablable
```

Comment:
I originally had it as that but then changed it because I thought I was breaking Ruby rules. But I'd rather break Ruby rules than grammar rules.... : )